poetry.lock

```diff
@@ -1,3 +1,22 @@
+[[package]]
```

Comment:
don't track poetry.lock the toml should be good enough

Reply:
Reverting the `.lock` file to its previous version and only changing the `.toml` seems to be causing errors during library installation